### PR TITLE
Update Wizard Examples to incorporate design discussions

### LINKED ISSUE #1367: Create Wizard variation
Please create a variation of Wizard based on the CCP designs. https://www.figma.com/file/qxdE6A2qswCVbW3R6Ti7KG/CCS-v0.09-(New-Navigation)?node-id=7848%3A45211

After we have the deploy link we will re-asses the template.

### PR DESCRIPTION
**Pull request by taysea**
[Preview](https://deploy-preview-1377--keen-mayer-a86c8b.netlify.app/templates/wizard)

#### What does this PR do?

Updates wizard examples to incorporate new wizard designs. All details noted below in "Design System updates".

#### Where should the reviewer start?
Any file.

#### What testing has been done on this PR?
Locally.

#### How should this be manually tested?
With deploy linked above.

#### What are the relevant issues?
Closes #1367 
#### Should this PR be mentioned in Design System updates?
- Wizard header has background of `background-contrast`
- Step titles use h1, size="small"
- Tighten up spacing between "Step 1 of X", Step Title, and Step description

#### Is this change backwards compatible or is it a breaking change?
Backwards compatible functionality, just visual changes.

### CODE REVIEW

**Comment by halocline** [comment]
@taysea is this ready for review?

File: aries-site/src/layouts/content/Example/BrowserWrapper.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ export const BrowserWrapper = ({ screen, ...rest }) => {
           direction="row"
           flex={false}
           gap="xsmall"
-          background="background-contrast"
+          background={{ color: 'border-weak' }}
```
Comment (halocline) [inline aries-site/src/layouts/content/Example/BrowserWrapper.js:25]: This seems odd to be using a "border" color for a background. Can we add a comment here explaining why?
```suggestion
          /* This Box represents a header of a browser application window. The selected background color 
           provides better contrast between the header and the examples being showcased. */
          background={{ color: 'border-weak' }}
```

File: aries-site/src/pages/templates/wizard.mdx [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
 <Example
-  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/wizard/WizardValidationExample.js"
+  code={[
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/wizard/WizardValidationExample.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/wizard/components/WizardHeader.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/wizard/components/StepHeader.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/wizard/components/StepContent.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/wizard/components/StepFooter.js',
+    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/wizard/components/CancellationLayer.js',
+  ]}
```
Comment (halocline) [inline aries-site/src/pages/templates/wizard.mdx:15]: 👍

File: aries-site/src/examples/templates/wizard/components/data.js [hunk 3]
```diff
@@ -0,0 +1,123 @@
+import { Text } from 'grommet';
+
+export const identityProvider = [
```
Comment (halocline) [inline aries-site/src/examples/templates/wizard/components/data.js:3]: I must be blind. Where are these objects being consumed?

**Review by halocline — COMMENTED** [review]
Looks great overall and nice clean ups. I had a couple small questions.

File: aries-site/src/layouts/content/Example/BrowserWrapper.js (hunk 1 above)
Comment (taysea) [reply aries-site/src/layouts/content/Example/BrowserWrapper.js:25]: Yes, sounds good.

File: aries-site/src/examples/templates/wizard/components/data.js (hunk 3 above)
Comment (taysea) [reply aries-site/src/examples/templates/wizard/components/data.js:3]: Whoops. These can be removed now, thanks. I had initially added them when the demo was aligned with CCS specific steps, but removed once I was making the demo more generic again. I'll remove. Thanks!

**Pull request merged**